app/assets/stylesheets/variables/_agaricus.scss

```diff
@@ -9,59 +26,78 @@ $xanthodermus_stain:   #D4A833;
-$MENU_BG_COLOR:               white;
-$MENU_HOT_FG_COLOR:           black;
-$MENU_HOT_BG_COLOR:           $xanthodermus_stain;
-$MENU_WARM_FG_COLOR:          black;
-$MENU_WARM_BG_COLOR:          $lilaceps_stain;
+// ==================================================
+// SECTION 2: SEMANTIC COLOR ASSIGNMENTS
+// ==================================================
+
+$PRIMARY_COLOR:                     $brasiliensis_gills_1;
+$SECONDARY_COLOR:                   $augustus_cap;
+$SUCCESS_COLOR:                     #70C070;
+$INFO_COLOR:                        $polyozellus_dark;
+$WARNING_COLOR:                     $xanthodermus_stain;
+$DANGER_COLOR:                      $lilaceps_stain;
+$LIGHT_COLOR:                       $campestris_cap;
+$DARK_COLOR:                        $cupreobrunneus_gills;
+
+// ==================================================
+// SECTION 3: UI ELEMENT COLORS
+// ==================================================
+
+// --------------------------------------------------
+// Body and Typography
+// --------------------------------------------------
+
+$BODY_FG_COLOR:                     $cupreobrunneus_gills;
+$BODY_BG_COLOR:                     $campestris_cap;
+
+// --------------------------------------------------
+// Links
+// --------------------------------------------------
+
+$LINK_WEIGHT:                       normal;
+$LINK_FG_COLOR:                     $brasiliensis_gills_1;
+$LINK_BG_COLOR:                     transparent;
+$LINK_VISITED_FG_COLOR:             $cupreobrunneus_gills;
+$LINK_VISITED_BG_COLOR:             transparent;
+$LINK_HOVER_FG_COLOR:               $brasiliensis_gills_2;
+$LINK_HOVER_BG_COLOR:               $campestris_cap;
+
+// --------------------------------------------------
+// Buttons
+// --------------------------------------------------
+
+$BUTTON_FG_COLOR:                   #000000;
+$BUTTON_BG_COLOR:                   #F6F6F6;
+$BUTTON_BORDER_COLOR:               #CCCCCC;
+$BUTTON_HOVER_FG_COLOR:             #000000;
+$BUTTON_HOVER_BG_COLOR:             #F6F6F6;
+$BUTTON_HOVER_BORDER_COLOR:         black;
+$BUTTON_ACTIVE_FG_COLOR:            $campestris_cap;
+$BUTTON_ACTIVE_BG_COLOR:            $brasiliensis_gills_2;
+$BUTTON_PRIMARY_FG_COLOR:           $campestris_cap;
+$BUTTON_PRIMARY_BG_COLOR:           $brasiliensis_gills_1;
+$BUTTON_PRIMARY_BORDER_COLOR:       $brasiliensis_gills_2;
+$BUTTON_PRIMARY_HOVER_FG_COLOR:     $campestris_cap;
+$BUTTON_PRIMARY_HOVER_BG_COLOR:     $brasiliensis_gills_2;
+$BUTTON_PRIMARY_HOVER_BORDER_COLOR: $cupreobrunneus_gills;
+$BUTTON_BORDER_WIDTH:               1px;
+$BUTTON_BORDER_STYLE:               solid;
+$BUTTON_BORDER_RADIUS:              5px;
+
+// --------------------------------------------------
+// Form Inputs
+// --------------------------------------------------
+
+$INPUT_FG_COLOR:                    #000000;
+$INPUT_BG_COLOR:                    #FFFFFF;
+$INPUT_BORDER_COLOR:                #CCCCCC;
```

Comment:
See above. This is defining white inputs with black text, but they should be dark IMO.